scripts/booster-integration-tests.sh

```diff
@@ -55,6 +55,9 @@ for dir in $(ls -d test-*); do
         "simplify")
             SERVER=$BOOSTER_DEV ./runDirectoryTest.sh test-$name $@
             ;;
+        "log-simplify-json")
+            SERVER="${KORE_RPC_BOOSTER} --log-format json -l Simplify --log-file test-$name/simplify-log.txt" ./runDirectoryTest.sh test-$name $@
+            ;;
```

Comment:
made several changes here, including adding the conversion from the original custom log levels tot he context filtering. Also renamed `--simplification-log-file` to just `--log-file`